common/src/main/java/com/revenuecat/purchases/common/utils.kt

```diff
@@ -12,10 +12,11 @@ import com.android.billingclient.api.BillingClient
 import com.android.billingclient.api.BillingResult
 import com.android.billingclient.api.Purchase
 import com.android.billingclient.api.PurchaseHistoryRecord
-import com.android.billingclient.api.SkuDetails
 import java.security.MessageDigest
 import java.util.Locale
 
+const val MICROS_MULTIPLIER = 1_000_000.0
```

Comment:
The reason I made it double is because when doing `productDetails?.priceAmountMicros?.div(MICROS_MULTIPLIER)` if MICROS_MULTIPLIER were an int, the decimal points on the division result would be removed.

This is also used when building a `ProductDetails` from a `SkuDetails`. By the way, I only extracted this because detekt was complaining about using a magic number. I don't think it's a big deal and can revert the extraction.